### DISCUSSION TITLE
Unable to close old mosh session screen in ipad

### DISCUSSION THREAD

**Discussion by fernandoflorez** [blinksh/blink#1907] [Support] [answered]
Hello,

2 or 3 months ago I opened a mosh session to a server that does not exist anymore. Now I’m unable to close that screen. It displays the “reconnecting” message on top but entering the keyboard shortcut does nothing. It doesn’t display the bottom menu to close it no matter what I try.

What can I do?

Thanks!

**Comment by carloscabanero** [comment]
Hey @fernandoflorez, thanks for reaching out. What version of Blink are you running? The contextual bar here should help you close the tab: https://docs.blink.sh/basics/navigation#blink-shell-gestures

**Reply by fernandoflorez** [reply]
Thank for your response. I'm running 17.1.0 . Contextual bar doesn't open on this specific session unfortunately.

**Reply by carloscabanero** [reply, marked as answer]
So a bunch of things that can be done in this case:
- Using the Close from the new three buttons on top of every window on the iPad should close the app without sessions being stored.
- Mosh has an escape sequence, you can see it at the top of the bar, usually `Ctrl-^ .` Pressing that should exit the session and take you back to Blink Shell.
- Cmd-w should close the tab.
- We never heard about the Contextual bar not showing up, but if you create a new tab and it is shown in there, then you should be able to press the Close button multiple times until you close everything.

**Reply by fernandoflorez** [reply]
CMD + W  worked! thanks
